### ISSUE TITLE
URL field on admin UI turns into physical path when it's external

### ISSUE THREAD

**Issue by dsvictorh** [keystonejs/keystone#2255]
This isn't a big deal but when you save an external URL using the URL field type the Admin UI strips it of its protocol. This is fine for the text inside the anchor but not for the actual href attribute as it turns into a physical address and ends up like this **http://server/keystone/theUrl** rather than what the actual URL should be **http://theUrl**

I think this is just that this field doesn't have a case for it specifically. Something like this on the mixins/columns jade could be the solution:

``` jade
else if col.type == 'url'
		+column_link(col.field.format(item), item.get(col.path), true)
```

**Comment by wmertens** [comment]
@dsvictorh Indeed that fixes it :)

**Comment by JedWatson** [comment]
Fixed by #2305
